exercises/concept/pretty-leaflet/.docs/instructions.md

```diff
@@ -4,54 +4,49 @@ Your acquaintance Erin needs to print thousands of handbills for multiple events
 
 ## 1. Create the class with a capitalized header
```

Comment:
```suggestion
## 1. Create a capitalized header
```
```suggestion
## 1. Create the class with a capitalized header
```

Reply:
Not sure why the interface is now "double suggesting".  Looks like some sort of bug.